CubicalBinarySystem.lagda

```diff
@@ -167,7 +167,7 @@ for the path constructors eqL, eqC and eqR, for which hcomp is used:
 path-construction : {ℓ : Level} {X : Type ℓ}
                     (x y : X)
                     (p : x ≡ y)
-                  → (i : I) → x ≡ p i
+                  → PathP (λ i → x ≡ p i) (refl ∙ refl) (p ∙ refl)
```

Comment:
There might be a more general horizontal composition function in the library that does this

Reply:
I think it is much clearer this way. If you wish, you can do the other ones you mentioned as well.

Reply:
I'll change them then merge, I couldn't find horizontal composition in the cubical library